packages/content-api/package.json

```diff
@@ -4,25 +4,34 @@
   "repository": "https://github.com/TryGhost/Ghost-SDKs/tree/master/packages/content-api",
   "author": "Ghost Foundation",
   "license": "MIT",
-  "main": "index.js",
+  "main": "dist/content-api.js",
+  "umd:main": "dist/content-api.umd.js",
+  "module": "lib/index.js",
+  "source": "lib/index.js",
   "scripts": {
     "dev": "echo \"Implement me!\"",
+    "pretest": "yarn build",
     "test": "NODE_ENV=testing mocha './test/**/*.test.js'",
-    "prebuild": "rm -rf build && mkdir build",
-    "build": "browserify -t varify -g es6ify -s ContentApi index.js > build/content-api.lib.js",
-    "lint": "eslint . --ext .js --cache",
+    "build": "rollup -c",
+    "lint": "eslint ./src --ext .js --cache",
```

Comment:
Should be swapped too `./lib` or follow https://github.com/TryGhost/Ghost-SDK/pull/31/commits/ca0f07afe86bd28a638e585731a6091a6863e28f to only ignore `dist`